packages/gatsby/cache-dir/production-app.js

```diff
@@ -28,6 +28,12 @@ setApiRunnerForLoader(apiRunner)
 
 navigationInit()
 
+window.browserLoc = {
```

Comment:
This is needed to decouple the expected location with the real location, and be able to modify it in `onClientEntry` calls.